packages/celery-library/src/celery_library/signals.py

```diff
@@ -26,21 +23,14 @@ def _init(startup_complete_event: threading.Event) -> None:
         loop = asyncio.new_event_loop()
         asyncio.set_event_loop(loop)
 
-        async def _setup_task_manager():
-            assert sender.app  # nosec
-            assert isinstance(sender.app, Celery)  # nosec
+        assert sender.app  # nosec
+        assert isinstance(sender.app, Celery)  # nosec
 
-            app_server.task_manager = await create_task_manager(
-                sender.app,
-                celery_settings,
-            )
-
-            set_app_server(sender.app, app_server)
+        set_app_server(sender.app, app_server)
 
         app_server.event_loop = loop
 
-        loop.run_until_complete(_setup_task_manager())
-        loop.run_until_complete(app_server.lifespan(startup_complete_event))
+        loop.run_until_complete(app_server.start_and_hold(startup_complete_event))
```

Comment:
we use `lifespan` everywhere. not sure that renaming makes sense, we need consistency. Please check with @pcrespov and @GitHK

Reply:
Connected to https://github.com/ITISFoundation/osparc-simcore/pull/8237/files#r2293556041